llvm/lib/Target/AMDGPU/AMDGPUAttributor.cpp

```diff
@@ -1408,8 +1433,14 @@ static bool runImpl(Module &M, AnalysisGetter &AG, TargetMachine &TM,
     }
   }
 
-  ChangeStatus Change = A.run();
-  return Change == ChangeStatus::CHANGED;
+  bool Changed = A.run() == ChangeStatus::CHANGED;
```

Comment:
Can this be more refined on waves per eu or flat workgroup size changing?

Reply:
I didn't follow your question. Can you expatiate it?